Makefile

```diff
@@ -60,6 +57,20 @@ update-resources:
 	./generate && \
 	rm generate && \
 	popd
-	
 
-.PHONY: bin default generate test vet bootstrap fmt fmtcheck update-resources
+# =============================================================================
+# Cermati specific command
+# =============================================================================
+VERSION := $(shell cat version)
+cermati-devserver:
+	@sh -c "/opt/vault/bin/vault server -dev -dev-root-token-id=root -dev-plugin-dir=./bin"
+
+# bin generates the releaseable binaries for this plugin
+cermati-bin: fmtcheck generate
+	@CGO_ENABLED=0 BUILD_TAGS='$(BUILD_TAGS)' XC_ARCH=amd64 XC_OS=linux XC_OSARCH=linux/amd64 sh -c "'$(CURDIR)/scripts/build.sh'"
+
+cermati-publish-s3: cermati-bin
+	aws --profile cermati s3 cp --recursive --exclude "*" --include "*.zip" pkg/ s3://com.cermati.infra.build.artifacts/vault-plugin/gcp-cermati/v$(VERSION)-release/
+# =============================================================================
+
+.PHONY: bin default generate test vet bootstrap fmt fmtcheck update-resources
```

Comment:
You should add `cermati-devserver`, `cermati-bin`, and `cermati-publish-s3` to .PHONY

Reply:
Noted. I'll update it in the next commit.